SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -557,39 +557,53 @@ void USpatialSender::SendAuthorityIntentUpdate(const AActor& Actor, VirtualWorke
 	}
 }
 
-void USpatialSender::SetAclWriteAuthority(const Worker_EntityId EntityId, const PhysicalWorkerName& DestinationWorkerId)
+void USpatialSender::SetAclWriteAuthority(
```

Comment:
Rewritten so as not to have any dependency on the static component view, as this allows for the the load balance enforcer and the sender to have a better defined API boundary. Not sure if this is the best change in the world, but it was nice to write.

Reply:
After all, on the GDK list of code authorship priorities, things that make Jacques feel good is top 3